### ISSUE TITLE
docker-compose installation manual

### ISSUE THREAD

**Issue by starlessboi** [Tiledesk/tiledesk-server#4]
is there any tutorial on using docker-compose to install tiledesk?
I can not use it, it gives errors
thanks

**Comment by andrealeo83** [comment]
Yes here you can find a docker compose guide: https://docs.tiledesk.com/installation/running-tiledesk-with-docker-compose
